cms/envs/common.py

```diff
@@ -898,6 +898,9 @@
     # programs support
     'openedx.core.djangoapps.programs',
 
+    # Catalog integration
```

Comment:
@schenedx we do, it'll be used to facilitate API calls when programs are moved to the catalog.